contracts/Diamond/facets/MarketFacet.sol

```diff
@@ -0,0 +1,332 @@
+pragma solidity 0.8.13;
+
+import "../libraries/appStorage.sol";
+import "../libraries/LibHelper.sol";
+import "../libraries/LibAccessCheck.sol";
+import "../../Tokens/V0.8.13/VTokens/VToken.sol";
+
```

Comment:
NetSpec comments each facets to explain pupose of the facets

Reply:
Added, 0a0df6067f7595709915dd32fae4fa4269517b65